src/patternfly/components/InlineEdit/examples/InlineEdit.md

```diff
@@ -0,0 +1,362 @@
+```hbs title=Single-inline-edit-(default)
+{{#> inline-edit inline-edit--id="single-inline-edit-example" inline-edit--type="form"}}
+  {{#> inline-edit-group inline-edit-group--modifier="pf-m-value-group"}}
+    {{#> inline-edit-value inline-edit-value--IsLabel="true"}}
+      Static value
+    {{/inline-edit-value}}
+    {{#> inline-edit-toggle}}
+      {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'id="' inline-edit--id '-edit-button" aria-label="edit" aria-labelledby="' inline-edit--id '-edit-button ' inline-edit--id '-label"')}}
+        <i class="fas fa-pencil-alt" aria-hidden="true" />
+      {{/button}}
+    {{/inline-edit-toggle}}
+  {{/inline-edit-group}}
+  {{#> inline-edit-group inline-edit-group--modifier="pf-m-input-group"}}
+    {{#> inline-edit-input}}
+      {{> form-control controlType="input" input="true" form-control--attribute='type="text" value="Static value" aria-label="Editable text input"'}}
+    {{/inline-edit-input}}
+    {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group"}}
+      {{> inline-edit-action-save inline-edit-action-save--IsIcon="true"}}
+      {{> inline-edit-action-cancel inline-edit-action-cancel--IsIcon="true"}}
+    {{/inline-edit-group}}
+  {{/inline-edit-group}}
+{{/inline-edit}}
+```
+
+```hbs title=Single-inline-edit-(active)
+{{#> inline-edit inline-edit--id="single-editable-example" inline-edit--type="form" inline-edit--modifier="pf-m-editable"}}
+  {{#> inline-edit-group inline-edit-group--modifier="pf-m-value-group"}}
+    {{#> inline-edit-value inline-edit-value--IsLabel="true"}}
+      Static value
+    {{/inline-edit-value}}
+    {{#> inline-edit-toggle}}
+      {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'id="' inline-edit--id '-edit-button" aria-label="edit" aria-labelledby="' inline-edit--id '-edit-button ' inline-edit--id '-label"')}}
+        <i class="fas fa-pencil-alt" aria-hidden="true" />
+      {{/button}}
+    {{/inline-edit-toggle}}
+  {{/inline-edit-group}}
+  {{#> inline-edit-group inline-edit-group--modifier="pf-m-input-group"}}
+    {{#> inline-edit-input}}
+      {{> form-control controlType="input" input="true" form-control--attribute='type="text" value="Static value" aria-label="Editable text input"'}}
+    {{/inline-edit-input}}
+    {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group"}}
+      {{> inline-edit-action-save inline-edit-action-save--modifier="pf-m-valid" inline-edit-action-save--IsIcon="true"}}
+      {{> inline-edit-action-cancel inline-edit-action-cancel--IsIcon="true"}}
+    {{/inline-edit-group}}
+  {{/inline-edit-group}}
+{{/inline-edit}}
+```
+
+```hbs title=Single-inline-edit-with-title-(default)
+{{#> inline-edit inline-edit--type="form" inline-edit--id="single-inline-edit-with-title-example"}}
+  {{#> inline-edit-group}}
+    {{#> title title--modifier="pf-m-xl" titleType="h2" title--attribute=(concat 'id="' inline-edit--id '-label"')}}
+      Single inline edit group
```

Comment:
From what I see in the design document it looks like this should have more space underneath and maybe be smaller, but its hard to tell. IMO it looks too big but let me know if there is a design spec to work from
<img width="457" alt="Screen Shot 2020-01-07 at 3 59 03 PM" src="https://user-images.githubusercontent.com/20118816/71929028-b74e2b00-3166-11ea-992a-7c99f239b9b1.png">

Reply:
Inline edit doesn't style or add margin/padding/spacing to anything other than items inside its groups, as their visibility is controlled by the component. It vertical spacing is controlled by a layout or parent component, such as `.pf-c-form` or `.pf-c-list`. In this case, I added support for `<dl>` styling, based on the design files, in the components workspace .css. That may make it confusing to view...